frontend/src/metabase/ui/components/inputs/MultiSelect/MultiSelect.styled.tsx

```diff
@@ -21,6 +21,11 @@ const VALUE_SIZES = {
   md: rem(28),
 };
 
+const RIGHT_SECTION_SIZES = {
```

Comment:
Let's add a repro for it. Cypress will error when you try to `click()` an element covered by another element.

Reply:
I tried adding an e2e test and it didn't help. Cypress can always click the button 
<img width="358" alt="Screenshot 2024-07-19 at 13 22 14" src="https://github.com/user-attachments/assets/182e320c-9e1c-416b-96e8-6f93603eae9e">

Reply:
I added a test that checks `getBoundingClientRect`

Reply:
Before
<img width="632" alt="Screenshot 2024-07-19 at 13 38 35" src="https://github.com/user-attachments/assets/273e30af-42d8-4e3d-8bb5-56c16e7c3500">

After
<img width="632" alt="Screenshot 2024-07-19 at 13 42 08" src="https://github.com/user-attachments/assets/2b38bb5c-dfb9-4e2b-ae6c-7894073c598f">